src/Integration.Vsix.UnitTests/Analysis/UserSettingsUpdaterTests.cs

```diff
@@ -124,52 +131,48 @@ public void SettingsChangeNotificationIsRaised()
     }
 }";
         var fileSystemMock = CreateMockFile(fileName, originalData);
-        var userSettingsProvider = new UserSettingsProvider(Substitute.For<ILogger>(), fileSystemMock, fileName);
-        var singleFileMonitorMock = CreateMockFileMonitor(fileName);
+        var settingsProvider = new UserSettingsProvider(Substitute.For<ILogger>(), fileSystemMock, fileName);
+        MockPath(fileName);
         int eventCount = 0;
         var settingsChangedEventReceived = new ManualResetEvent(initialState: false);
 
-        var testSubject = CreateTestSubject(singleFileMonitorMock, userSettingsProvider: userSettingsProvider);
-        userSettingsProvider.UserSettings.RulesSettings.Rules.Count.Should().Be(0); // sanity check of setup
+        var userSettingsUpdater = CreateUserSettingsUpdater(singleFileMonitorFactory, slCoreRuleSettings, settingsProvider);
```

Comment:
Should this still be here even though we have TestInitialize now?

Reply:
Yes, because the settings provider is not the mocked one (as in the TestInitialitze), it is a real instance/